src/app/view/render2D/featureRenderer2D.js

```diff
@@ -118,7 +118,7 @@ export function renderText(feature) {
  * @param feature
  * @return {*}
  */
-export function renderFeature(feature, key = null) {
+export function renderFeature(feature, key) {
```

Comment:
Yeah this ties back to the whole don't pass `undefined` business in `component` and `connection`

Reply:
Yes